apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -99,15 +79,10 @@ export class AuthorizationService extends BasePermissionManager {
 	}
 
 	async getUserWithPermissions(userId: EntityId): Promise<User> {
-		try {
-			const userWithPermissions: User = (await this.loader.loadEntity(
-				AllowedAuthorizationEntityType.User,
-				userId
-			)) as User;
-
+		const userWithPermissions = await this.loader.loadEntity(AllowedAuthorizationEntityType.User, userId);
+		if (userWithPermissions instanceof User) {
 			return userWithPermissions;
-		} catch (err) {
-			throw new ForbiddenException(err);
 		}
+		throw new ForbiddenException();
```

Comment:
Willst du den Type Guard in eigene Helper Funktion auslagern? Den brauchen wir evtl. auch an anderen Stellen noch. Bzw. könntest du ihn dann auch separat testen.